core-bundle/src/Command/Backup/BackupStreamContentCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command\Backup;
+
+use Contao\CoreBundle\Doctrine\Backup\BackupManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\StreamOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @internal
+ */
+class BackupStreamContentCommand extends Command
+{
+    protected static $defaultName = 'contao:backup:stream-content';
+
+    protected BackupManager $backupManager;
+
+    public function __construct(BackupManager $backupManager)
+    {
+        $this->backupManager = $backupManager;
+
+        parent::__construct();
+    }
+
```

Comment:
Should we hide this command in the list as it's rather technical?

```suggestion

    protected function configure(): void
    {
        $this->setHidden(true);
    }
    
```

Reply:
Agreed, done in b2cd7a26c45224879797e51a950be57a5569b8ce